bundles/SeoBundle/src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,99 @@
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+
+namespace Pimcore\Bundle\SeoBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+
+    /**
+     * @inheritDoc
+     */
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('pimcore_seo');
+
+        $rootNode = $treeBuilder->getRootNode();
+        $rootNode->addDefaultsIfNotSet();
+
+        $this->addSitemapGenerators($rootNode);
+        $this->addRedirectsConfig($rootNode);
+
+        return $treeBuilder;
+    }
+
+    private function addSitemapGenerators(ArrayNodeDefinition $rootNode):void {
+        $rootNode
+            ->children()
+                ->arrayNode('sitemaps')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('generators')
+                            ->useAttributeAsKey('name')
+                            ->prototype('array')
+                                ->beforeNormalization()
+                                    ->ifString()
+                                    ->then(function ($v) {
+                                        return [
+                                            'enabled' => true,
+                                            'generator_id' => $v,
+                                            'priority' => 0,
+                                        ];
+                                    })
+                                ->end()
+                                ->addDefaultsIfNotSet()
+                                ->canBeDisabled()
+                                ->children()
+                                    ->scalarNode('generator_id')
+                                        ->cannotBeEmpty()
+                                    ->end()
+                                    ->integerNode('priority')
+                                        ->defaultValue(0)
+                                    ->end()
+                                ->end()
+                            ->end()
+                        ->end()
+                    ->end()
+                ->end()
+            ->end()
+        ->end();
+    }
+
+    private function addRedirectsConfig(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('redirects')
+                ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('status_codes')
+                            ->info('List all redirect status codes.')
+                                ->prototype('scalar')
+                            ->end()
+                        ->end()
+                        ->booleanNode('create_redirects')
```

Comment:
this should be documented on Redirects page

Reply:
But it is not only used for documents_move. It is a general flag if you want to create redirects. As I mentioned in the daily, just because you install the SeoBundle, does not mean, that you automatically want redirects to be created.

Reply:
Still this is quite generic and hard to understand. It could be something like `auto_create_redirects` and we should also add description to the config node.